frameworks/digital-outcomes-and-specialists-4/questions/declaration/mitigatingFactors3.yml

```diff
@@ -2,9 +2,16 @@ name: Mitigating factors for Modern Slavery reporting requirements
 question: >
   If your organisation does not meet the requirements, please explain why.
 type: textbox_large
+max_length_in_words: 500
 
 hidden: true
 validations:
-  - 
+  -
     name: answer_required
-    message: You need to answer this question
+    message: You need to answer this question because you answered ‘yes’ to question [[modernSlaveryReportingRequirements]].
```

Comment:
Did we just not mention this question's dependency before?

Reply:
I thought I'd make it consistent with the other validation error messages (on `mitigatingFactors` and `mitigatingFactors2`)...